dj/api/nodes.py

```diff
@@ -259,21 +270,55 @@ def delete_a_node(name: str, *, session: Session = Depends(get_session)):
         downstream.current.status = NodeStatus.INVALID
         session.add(downstream)
 
-    # If the node is a dimension, find all columns that
-    # are linked to this dimension and remove the link
-    if node.type == NodeType.DIMENSION:
-        columns = (
-            session.exec(select(Column).where(Column.dimension_id == node.id))
-            .unique()
-            .all()
+    now = datetime.utcnow()
+    node.deactivated_at = UTCDatetime(
+        year=now.year,
+        month=now.month,
+        day=now.day,
+        hour=now.hour,
+        minute=now.minute,
+        second=now.second,
+    )
+    session.add(node)
+    session.commit()
```

Comment:
Does this call make all the child nodes `invalid` or is it somewhere else?

Reply:
That logic already existed and can be found in these two helper functions: https://github.com/DataJunction/dj/blob/main/dj/api/helpers.py#L394-L474

Reply:
Oh sorry that was a link to the logic that revalidates the children after a node is re-activated. The logic that makes all the child nodes `invalid` also already exists and is here: https://github.com/DataJunction/dj/blob/main/dj/api/nodes.py#L256-L260